q2_metadata/_random.py

```diff
@@ -14,14 +14,24 @@
 def shuffle_groups(metadata: qiime2.CategoricalMetadataColumn,
                    n_columns: int = 3,
                    column_name_prefix: str = 'shuffled.grouping.',
-                   column_value_prefix: str = 'fake.group.') -> pd.DataFrame:
+                   # Suggest renaming this variable to sample_value_prefix for
```

Comment:
This is a good point, I agree that we should name this something a bit more clear - will think about this and circle back! @gregcaporaso @cherman2 any thoughts on what would be the most clear?